keel-actuator/src/test/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuatorTests.kt

```diff
@@ -284,6 +284,12 @@ internal class ResourceActuatorTests : JUnit5Minutests {
 private fun <T : Iterable<E>, E> Assertion.Builder<T>.second(): Assertion.Builder<E> =
   get("second element %s") { toList()[1] }
 
-internal data class DummyResource(val state: String)
-
-internal data class DummyResourceSpec(val state: String)
+internal data class DummyResource(
+  val state: String,
+  val data: String = "some data"
+)
```

Comment:
Curious what the value is of the additional field here?

Reply:
with this dummy resource we calculate name based on `state`. When you change the state, it changes the name, which now will just persist a new resource. So, I had to add another field so that we could properly test an update situation.

Reply:
Gotcha. That makes sense.